tests/test_utils/logging_command_executor.py

```diff
@@ -41,11 +41,13 @@ def execute_cmd(self, cmd, silent=False, cwd=None, env=None):
             ) as process:
                 output, err = process.communicate()
                 retcode = process.poll()
-                self.log.info("Stdout: %s", output)
-                self.log.info("Stderr: %s", err)
                 if retcode:
                     self.log.error("Error when executing %s", " ".join(shlex.quote(c) for c in cmd))
-                return retcode
+                    raise AirflowException(
+                        f"Retcode {retcode} for '{' '.join(cmd)}' with stdout: {output}, stderr: {err}"
+                    )
```

Comment:
```suggestion
                    commands = " ".join(cmd)
                    raise AirflowException(
                        f"Retcode {retcode} for '{commands}' with stdout: {output}, stderr: {err}"
                    )
```

I needed to reread several times to properly parse the previous code mentally…

Reply:
Such part of code is existed already. I took it from there